src/Bindings/PluginLua.cpp

```diff
@@ -113,7 +114,7 @@ bool cPluginLua::Load(void)
 	bool HasInfoLua = false;
 	for (AStringVector::const_iterator itr = Files.begin(), end = Files.end(); itr != end; ++itr)
 	{
-		if (itr->rfind(".lua") != AString::npos)
+		if (itr->rfind(LUA_FILE_SUFFIX) == itr->size() - LUA_FILE_SUFFIX.size())
```

Comment:
Why not just compare the extension (from the last dot) to the match string? Then you won't need the superfluous static const variable at all.